cloud-interfaces/api/cloudservers-api.rst

```diff
@@ -115,16 +115,16 @@ for tag, status, image, flavor, and type.
 Learn about Cloud Servers in API documentation
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 In the
-:rax-api:`API cross-reference <>`,
+:rax-docs: `Developer documentation <>`.
```

Comment:
This link is broken - it looks like this in the output: :rax-docs: Developer documentation <>